packages/eslint-config/package.json

```diff
@@ -1,50 +1,50 @@
-{
-  "name": "@unocss/eslint-config",
-  "version": "0.58.9",
-  "description": "ESLint config for UnoCSS",
-  "author": "Anthony Fu <anthonyfu117@hotmail.com>",
-  "license": "MIT",
-  "funding": "https://github.com/sponsors/antfu",
-  "homepage": "https://github.com/unocss/unocss/tree/main/packages/eslint-config#readme",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/unocss/unocss",
-    "directory": "packages/eslint-config"
-  },
-  "bugs": {
-    "url": "https://github.com/unocss/unocss/issues"
-  },
-  "keywords": [
-    "eslint-config",
-    "eslint"
-  ],
-  "sideEffects": false,
-  "exports": {
-    ".": {
-      "types": "./dist/index.d.ts",
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.cjs"
-    },
-    "./flat": {
-      "types": "./dist/flat.d.ts",
-      "import": "./dist/flat.mjs",
-      "require": "./dist/flat.cjs"
-    }
-  },
-  "main": "dist/index.cjs",
-  "module": "dist/index.mjs",
-  "types": "dist/index.d.ts",
-  "files": [
-    "dist"
-  ],
-  "engines": {
-    "node": ">=14"
-  },
-  "scripts": {
-    "build": "unbuild",
-    "stub": "unbuild --stub"
-  },
-  "dependencies": {
-    "@unocss/eslint-plugin": "workspace:*"
-  }
-}
+{
+  "name": "@unocss/eslint-config",
+  "version": "0.58.9",
+  "description": "ESLint config for UnoCSS",
+  "author": "Anthony Fu <anthonyfu117@hotmail.com>",
+  "license": "MIT",
+  "funding": "https://github.com/sponsors/antfu",
+  "homepage": "https://github.com/unocss/unocss/tree/main/packages/eslint-config#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/unocss/unocss",
+    "directory": "packages/eslint-config"
+  },
+  "bugs": {
+    "url": "https://github.com/unocss/unocss/issues"
+  },
+  "keywords": [
+    "eslint-config",
+    "eslint"
+  ],
+  "sideEffects": false,
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",

```

Comment:
remove types here, we're masquerading CJS entry: dual ESM/CJS shouldn't have types

Reply:
should remove types also in postcss and webpack